test/Makefile

```diff
@@ -97,6 +97,7 @@ QUIET=@
 export RESULTS_DIR=test_results
 export MODEL
 export REQUIRED_ARGS=
+BUILD=release
```

Comment:
Yep, I had to do quite a few try to figure out a working setup and in the one you reviewed I temporarily removed the new changes with `$(BUILD)`